deploy/legionPipeline.groovy

```diff
@@ -103,6 +103,30 @@ def deployLegion() {
     }
 }
 
+def UpdateTLSCert() {
+    withCredentials([
+    file(credentialsId: "vault-${env.param_profile}", variable: 'vault')]) {
+        withAWS(credentials: 'kops') {
+            wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+                docker.image("${env.param_docker_repo}/k8s-ansible:${env.param_legion_version}").inside("-e HOME=/opt/legion/deploy -v ${WORKSPACE}/deploy/profiles:/opt/legion/deploy/profiles -u root") {
+                    stage('Reissue TLS Certificates') {
+                        sh """
+                        cd ${ansibleHome} && \
+                        ansible-playbook update-tls-certificate.yml \
+                        ${ansibleVerbose} \
+                        --vault-password-file=${vault} \
+                        --extra-vars "profile=${env.param_profile} \
+                        vault_pass=${vault} \
+                        docker_repo=${env.param_docker_repo} \
```

Comment:
Please remove unused arguments (docker_repo and legion_version)

Reply:
done